parseargs.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import sys
+
+import pytest
+
+
+class ParseArgs:
```

Comment:
I think it makes more sense for this to be the CLI.

Don't import or call this from model.py, do it the other way around.

Also looks like the predict subcommand doesn't work. Come up with some way of saving and loading a model (and add them as functions or methods in model.py) and call those methods from here.

This becomes the CLI entry point to training a model.

---

If you write other CLIs in the future, you might want to look up alternatives to argparse that are a bit easier to use. [Docopt](https://github.com/docopt/docopt) is popular. I wrote [cli](https://github.com/cmcaine/cli), which I obviously prefer.

You don't need to change it now, but the normal way to handle subcommands is with "subparsers". I use them in `generate_parser_obj()` in cli.py

Or you could just write something like this:

```python
class StopItCLI:
    def build(path_to_data, text_column_name, label_column_name, *, 
            vocab_size=10000, batch_size=512, ...):
        Model().build(path_to_data, text_column_name, label_column_name,
                vocab_size, ...)

    def predict(sentence):
        pre_fitted_model = whatever
        print(Model().predict(sentence, pre_fitted_model, ...))

import cli
cli(StopItCLI)()
```

Reply:
Will go with what we've got for now but look at this for next time!